photon-core/src/main/java/org/photonvision/common/configuration/ConfigManager.java

```diff
@@ -116,7 +117,7 @@ private void translateLegacyIfPresent(Path folderPath) {
                 }
 
                 // So we can't save the old config, and we couldn't copy the folder
-                // But we've loaded the config. So just try to delete the directory so we don't try to load
+                // But we've loaded the config. So just try to delete the directory, so we don't try to load
```

Comment:
This comma doesn't actually make this make any more sense (arguably less grammatically correct)